client/src/modules/shopping-list/index.jsx

```diff
@@ -120,20 +120,18 @@ class ShoppingList extends Component {
           <ProductsContainer archived products={archiveList} />
         </div>
         {showDialogBox && (
-          <ModalBox>
-            <DialogBox
-              onCancel={this.hideDialogBox}
-              onConfirm={this.deleteListHandler(listId)}
-              message="Do you really want to delete the list?"
-            />
-          </ModalBox>
+          <DialogBox
+            message="Do you really want to delete the list?"
+            onCancel={this.hideDialogBox}
+            onConfirm={this.deleteListHandler(listId)}
+          />
         )}
-        {showUpdateFrom && (
-          <ModalBox onClose={this.hideUpdateForm}>
+        {showUpdateForm && (
+          <ModalBox onCancel={this.hideUpdateForm}>
```

Comment:
this modal box would need to be refactored so that it is used inside components not outside of them